runcommandonset/tests/runcommandonset.get.tests.ps1

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+Describe 'tests for runcommandonset get' {
+    BeforeAll {
+        $oldPath = $env:DSC_RESOURCE_PATH
+        $env:DSC_RESOURCE_PATH = Join-Path $PSScriptRoot ".."
+    }
+
+    AfterAll {
+        $env:DSC_RESOURCE_PATH = $oldPath
+    }
+
+    It 'Input can be sent to the resource via stdin json' {
```

Comment:
You really only need either hte JSON or YAML test as it's really exercising `dsc` as `dsc` always passes JSON to the resource.